teuthology/openstack/__init__.py

```diff
@@ -617,6 +617,59 @@ def main(self):
                 self.teardown()
         return exit_code
 
+    def upload_yaml_file(self, fp):
```

Comment:
Since this method is only used from within its object it could be marked protected ```_upload_yaml_file```